packages/gatsby-theme-store/src/components/Auth/Providers/ExternalProvider/index.tsx

```diff
@@ -28,6 +28,7 @@ const ExternalProvider: FC<AuthProviderComponentProps> = ({
   const startLogin = useStartLogin()
   const [state, setState] = useState<string>('initial')
   const variant = `externalOAuth.${providerName}.${v}`
+  const nativeAuthProviders = ['Google', 'Facebook']
```

Comment:
move this to the global scope in a `Set` so the check `includes` functions takes O(1) and we don't waste heap memory creating this array every time this component renders